spec/lib/api/v3/work_packages/base_contract_spec.rb

```diff
@@ -28,7 +28,7 @@
 
 require 'spec_helper'
 
-describe ::API::V3::WorkPackages::WorkPackageContract do
+describe ::API::V3::WorkPackages::BaseContract do
```

Comment:
haha, apparently the specs on everything that you removed from the contract were incomplete...

or they are red now ^^